extensions/query-store/src/common/utils.ts

```diff
@@ -74,3 +74,8 @@ export function createSplitView(view: azdata.ModelView, firstComponent: azdata.C
 
 	return splitview;
 }
+
+export enum ConfigInfo {
```

Comment:
This enum is just required to figure out distinction between the two reports to add Time Interval component appropriately. With a little duplication of code, it can be removed. Thoughts on which track to take?

Overall Resource Consumption:
![image](https://github.com/microsoft/azuredatastudio/assets/57200045/219d0825-b213-4b43-b3c8-1df33ffa87f5)

Top Resource Consuming Queries:
![image](https://github.com/microsoft/azuredatastudio/assets/57200045/d0ed92ad-a501-441a-a254-9b2688a93bc4)